AutomatedTesting/Gem/PythonTests/atom_renderer/atom_hydra_scripts/hydra_GPUTest_LightComponent.py

```diff
@@ -0,0 +1,275 @@
+    directional_light_component = azlmbr.editor.EditorComponentAPIBus(
+        azlmbr.bus.Broadcast, 'GetComponentOfType', directional_light.id, directional_light_component_type
+    ).GetValue()
+    editor.EditorComponentAPIBus(bus.Broadcast, "DisableComponents", [directional_light_component])
+    general.idle_wait(0.5)
+
+    # Disable "Global Skylight (IBL)" and "HDRi Skybox" components for the "global_skylight" entity
+    global_skylight_entity_id = hydra.find_entity_by_name("global_skylight")
+    global_skylight = hydra.Entity(name='global_skylight', id=global_skylight_entity_id)
+    global_skylight_component_type = azlmbr.editor.EditorComponentAPIBus(
+        azlmbr.bus.Broadcast, 'FindComponentTypeIdsByEntityType', ["Global Skylight (IBL)"], 0)[0]
+    global_skylight_component = azlmbr.editor.EditorComponentAPIBus(
+        azlmbr.bus.Broadcast, 'GetComponentOfType', global_skylight.id, global_skylight_component_type
+    ).GetValue()
+    editor.EditorComponentAPIBus(bus.Broadcast, "DisableComponents", [global_skylight_component])
+    hdri_skybox_component_type = azlmbr.editor.EditorComponentAPIBus(
+        azlmbr.bus.Broadcast, 'FindComponentTypeIdsByEntityType', ["HDRi Skybox"], 0)[0]
+    hdri_skybox_component = azlmbr.editor.EditorComponentAPIBus(
+        azlmbr.bus.Broadcast, 'GetComponentOfType', global_skylight.id, hdri_skybox_component_type
+    ).GetValue()
+    editor.EditorComponentAPIBus(bus.Broadcast, "DisableComponents", [hdri_skybox_component])
+    general.idle_wait(0.5)
+
+    # Create a "spot_light" entity with "Light" component using Light Type of "Spot (disk)"
+    spot_light_entity_name = "spot_light"
+    light_component = "Light"
+    spot_light = hydra.Entity(f"{spot_light_entity_name}")
+    spot_light.create_entity(math.Vector3(0.7, -2.0, 1.0), [light_component])
+    general.log(
+        f"{spot_light_entity_name}_test: Component added to the entity: "
+        f"{hydra.has_components(spot_light.id, [light_component])}")
+    rotation = math.Vector3(DEGREE_RADIAN_FACTOR * 300.0, 0.0, 0.0)
+    azlmbr.components.TransformBus(azlmbr.bus.Event, "SetLocalRotation", spot_light.id, rotation)
+    light_component_type = atom_component_helper.attach_component_to_entity(spot_light.id, "Light")
+    light_type_property_path = 'Controller|Configuration|Light type'
+    spot_disk_light_type = atom_component_helper.LIGHT_TYPES['spot_disk']
+    editor.EditorComponentAPIBus(
+        azlmbr.bus.Broadcast,
+        'SetComponentProperty',
+        light_component_type,
+        light_type_property_path,
+        spot_disk_light_type
+    )
+
+    general.idle_wait(1.0)
+    screenshot_utils.take_screenshot_game_mode("SpotLight_1", spot_light_entity_name)
+
+    # Change default material of ground plane entity and take screenshot
+    ground_plane_entity_id = hydra.find_entity_by_name("ground_plane")
+    ground_plane = hydra.Entity(name='ground_plane', id=ground_plane_entity_id)
```

Comment:
this seems like an unnecessary step since everything then uses ground_plane.id which is actually already available as ground_plane_entity_id. either way is fine though

Reply:
just going to leave it as is for this PR